opapi/src/generators/client.ts

```diff
@@ -41,14 +41,6 @@ ${operations.map((operation) => generateMethod(operation)).join('\n')}
 }
 
 ${operations.map((operation) => generatePropsType(operation)).join('\n')}
-
-function getError(err: Error) {
-  if (axios.isAxiosError(err)) {
-    return errorFrom(err.response?.data)
```

Comment:
This was causing the error received in the client to have no message when the backend returns an HTTP error response with no body (like in the case of having an expired session).